--- ./configure	2008-05-05 14:36:16.000000000 +1000
+++ mk12.0.2/configure	2008-05-05 14:32:57.000000000 +1000
@@ -478,9 +478,9 @@
 fi
 # Find GNU Plotutils
 printf "checking GNU plotutils in $USR_LIB and $USR_INC ... "
-if [    -f $USR_INC/plot.h ]
+if [    -f $USR_INC/plot.h -o  -f $SYS_INC/plot.h ]
 then
-	if [    -f $USR_LIB/libplot.so -o -f $USR_LIB/libplot.a	-o -f $USR_LIB/libplot.dylib	]
+	if [    -f $USR_LIB/libplot.so -o -f $USR_LIB/libplot.a	-o -f $USR_LIB/libplot.dylib -o  -f $SYS_LIB/libplot.so	]
 	then
 		libplot=yes
 		echo "yes"
@@ -494,9 +494,9 @@
 fi
 # Find GSL
 printf "checking GNU scientific library in $USR_LIB ... "
-if [    -f $USR_LIB/libgsl.so -o -f $USR_LIB/libgsl.a -o -f $USR_LIB/libgsl.dylib 	]
+if [    -f $USR_LIB/libgsl.so -o -f $USR_LIB/libgsl.a -o -f $USR_LIB/libgsl.dylib -o -f  $SYS_LIB/libgsl.so 	]
 then
-	if [    -f $USR_LIB/libgslcblas.so -o -f $USR_LIB/libgslcblas.a -o -f $USR_LIB/libgslcblas.dylib	]
+	if [    -f $USR_LIB/libgslcblas.so -o -f $USR_LIB/libgslcblas.a -o -f $USR_LIB/libgslcblas.dylib -o -f $SYS_LIB/libgslcblas.so 	]
 	then
 		libgsl=yes
 		echo "yes"
@@ -571,6 +571,12 @@
 	echo "SQL_DIR=$USR_DIR">>$BUILD_FILE
 	echo "SQL_LIBS=-lpq" 1>>$BUILD_FILE
 	echo "SQL_FLAG=-DENABLE_POSTGRESQL" 1>>$BUILD_FILE
+elif [ -f $SYS_LIB/libpq.so -o -f $SYS_LIB/libpq.a -o -f $SYS_LIB/libpq.dylib	]
+then
+	echo "yes"
+	echo "SQL_DIR=`dirname $SYS_LIB`">>$BUILD_FILE
+	echo "SQL_LIBS=-lpq" 1>>$BUILD_FILE
+	echo "SQL_FLAG=-DENABLE_POSTGRESQL" 1>>$BUILD_FILE
 else
 	echo "no"
 fi
@@ -867,6 +873,9 @@
 elif [ -f $USR_LOC/pgsql/include/pg_config.h ]
 then
 	echo "#define A68_PG_VERSION \"`grep '#define PG_VERSION ' $USR_LOC/pgsql/include/pg_config.h | sed 's/#define[A-Z_ ]*//'`\"" | sed 's/\"\"/\"/g' 1>>$VERSION_FILE
+elif [ -f $SYS_INC/pgsql/include/pg_config.h ]
+then
+	echo "#define A68_PG_VERSION \"`grep '#define PG_VERSION ' $SYS_INC/pgsql/include/pg_config.h | sed 's/#define[A-Z_ ]*//'`\"" | sed 's/\"\"/\"/g' 1>>$VERSION_FILE
 fi
 #
 # Build target if required
@@ -896,15 +905,15 @@
 fi
 echo ""
 echo "configured $PACKAGE for `uname -rs`"
-if [ $libplot = "yes" ]
+if [ "$libplot" = "yes" ]
 then
 	echo "$PACKAGE will compile with support for the GNU plotting utilities"
 fi
-if [ $libgsl = "yes" ]
+if [ "$libgsl" = "yes" ]
 then
 	echo "$PACKAGE will compile with support for the GNU scientific library"
 fi
-if [ $libpostgresql = "yes" ]
+if [ "$libpostgresql" = "yes" ]
 then
 	echo "$PACKAGE will compile with support for Postgresql"
 fi
